### PR TITLE
SDPA-345 : Drag rather than click the POI circle

### PR DESCRIPTION
**Pull request by naomiaro**
POI initially updates on mousedown, you can drag the circle to adjust, then it will update again on mouse up/leave.

### CODE REVIEW

**Comment by pavlovicnemanja** [comment]
:+1:

File: scripts/superdesk/upload/image-crop-directive.js [hunk 1]
```diff
@@ -293,27 +295,62 @@
...
+                    function leaveOverlay(e) {
+                        e.preventDefault();
+                        updatePOI(e.offsetX, e.offsetY);
                         $rootScope.$broadcast('poiUpdate', scope.point);
+                        overlay.onmousemove = null;
+                        overlay.onmouseup = null;
+                        overlay.onmouseleave = null;
+                    }
+
+                    overlay.addEventListener('mousedown', function(e) {
```
Comment (petrjasek) [inline scripts/superdesk/upload/image-crop-directive.js]: this listener should be removed as well, otherwise there is a memory leak

Comment (naomiaro) [inline scripts/superdesk/upload/image-crop-directive.js]: ah yes, forgot about scope destroy

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**